python/taichi/lang/ast/ast_transformer.py

```diff
@@ -139,7 +164,11 @@ def build_assign_unpack(ctx, node_target, values, is_static_assign):
         return None
 
     @staticmethod
-    def build_assign_basic(ctx, target, value, is_static_assign):
+    def build_assign_basic(ctx,
+                           target,
+                           value,
+                           is_static_assign,
+                           build_target=True):
```

Comment:
`build_target` can be avoided with some rearrangements, refer to the comment above~